docs/4-Transition/lessons.md

```diff
@@ -54,6 +54,21 @@ shared_configuration:
 
 Now that the filesystem has been set to read/write, DynaTrace will no longer crash the pods.
 
+:::warning
+The team continued to experience issues with DynaTrace and FileNet IER deployment. As of writing, the FileNet Dev team is working on propagating the 5.5.11 read/write filesystem to IER. 
+:::
+
+The team had to use an alternative solution for IER. This included modifying a DaemonSet. A DaemonSet ensures that all (or some) nodes in a Kubernetes cluster run a copy of a specific pod. Unlike typical deployments where you specify the desired number of replicas, a DaemonSet ensures one pod instance per node. These pods are typically used for tasks like monitoring, logging, or other system-level services that should run on every node. We used the following command:
+
+```
+kubectl -n YOUR_NAMESPACE patch daemonset YOUR_DAEMONSET_NAME -p '{"spec": {"template": {"spec": {"nodeSelector": {"non-existing": "true"}}}}}'
+```
+:::note
+The previous command is a template. You need to modify `YOUR_NAMESPACE` and `YOUR_DAEMONSET_NAME` with your DynaTrace Namespace and DaemonSet name. 
```

Comment:
NAMESPACE should be clarified as to being the namespace where the dynatrace agent is deployed. This might be different from where filenet has been deployed. Let's add some notes here to specify that the user needs to determine the appropriate namespace.

Reply:
Updated awaiting further feedback.